airbyte-webapp/src/components/JobItem/components/JobLogs.tsx

```diff
@@ -23,9 +27,17 @@ const JobLogs: React.FC<IProps> = ({ id, jobIsFailed }) => {
   const job = useGetJob(id);
   const debugInfo = useGetDebugInfoJob(id);
 
-  const [attemptNumber, setAttemptNumber] = useState<number>(
-    job.attempts.length ? job.attempts.length - 1 : 0
-  );
+  const { hash } = useLocation();
+  const [attemptNumber, setAttemptNumber] = useState<number>(() => {
```

Comment:
As we now have attempt number in hash - do we really need to use state to store selected number? Can't we just pick the one from hash always? So when user opens some attempt - we already modify url and he could copy-paste it if necessary

Reply:
That wouldn't work, since you could have multiple Logs open. If we would want that we'd need to store the selected attempt state for ALL logs are are currently open in the hash, which imho goes a bit beyond what's needed here. Also I currently don't sync the state automatically to the hash, on purpose, since having the button to explicitly copy it, instead of implicitly copy the current URL after you switched to a specific attempt (and nowhere else afterwards), makes it easier for support to ask for that from the users.

Reply:
Ah, indeed. As we could open different logs open - it doesn't make that much sense to reflect that info into url on the fly.